mlflow/keras.py

```diff
@@ -195,15 +195,16 @@ def _load_pyfunc(path):
         with graph.as_default():
             with sess.as_default():  # pylint:disable=not-context-manager
                 K.set_learning_phase(0)
-                m = _load_model(path)
+                m = _load_model(path, *args, **kwargs)
         return _KerasModelWrapper(m, graph, sess)
     else:
         raise Exception("Unsupported backend '%s'" % K._BACKEND)
 
 
-def load_model(model_uri):
+def load_model(model_uri, *args, **kwargs):
```

Comment:
pytorch only passes along `**kwargs` here. This makes sense to me, since relying on positional params via a few layers of indirection seems a bit sketchy. Is there a strong reason to pass through `*args` as well?

Reply:
No.